builder/tart/builder.go

```diff
@@ -34,6 +34,7 @@ type Config struct {
 
 	CpuCount        uint8         `mapstructure:"cpu_count"`
 	CreateGraceTime time.Duration `mapstructure:"create_grace_time"`
+	BootWait        time.Duration `mapstructure:"boot_wait"`
```

Comment:
I don't think we need to add anything here since the `BootWait` is already present in the `builder.hcl2spec.go` by the virtue of squashing the `bootcommand.VNCConfig` onto `Config`.

Moreover, the `BootWait` is already parsed as `time.Time` and is available from `b.config.VNCConfig.BootWait`.